server/functions/package-lock.json

```diff
@@ -1,4093 +0,0 @@
-{
```

Comment:
Shouldn't `package-lock.json` be included in the repo @rlnsy ?

Reply:
Nope. Since we're using yarn it actually produces a warning if it's there. I didn't recall deleting it in this branch but if its okay with you I'll leave it in since it's something I've been meaning to do.